services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/__init__.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import Field
+from settings_library.base import BaseCustomSettings
+
+from .egress_proxy import EgressProxySettings
+from .proxy import DynamicSidecarProxySettings
+from .scheduler import DynamicServicesSchedulerSettings
+from .sidecar import DynamicSidecarSettings
+
+
+class DynamicServicesSettings(BaseCustomSettings):
```

Comment:
Q: you do not need the envs in `settings_library.application.BaseApplicationSettings` ??

Reply:
This is correct. This "submodule" does not require them.
The previous inheritance structure was not altered. As said, I was just moving them.